src/components/tabs/themes/tabs.base.scss

```diff
@@ -1,116 +1,155 @@
 @use 'styles/common/component';
 @use 'styles/utilities' as *;
-
-$tabs-animation-function: cubic-bezier(.35, 0, .25, 1);
+@use 'tabs.common' as *;
+@use 'sass:map';
 
 :host {
-    display: flex;
-    flex-direction: column;
+    // Scroll buttons
+    --scroll-btn-size: #{rem(48px)};
+    --tabs-width: 100%;
+    --tab-max-width: #{rem(360px)};
+    --tab-min-width: #{rem(90px)};
+    --_border-size: #{rem(1px)};
```

Comment:
It may be prudent to include these properties as part of the component theme schema as they need to be in sync with Ignite UI for Angular. Also, I see that some of the properties are used only once, I don't see why they need to be defined is CSS variables. I prefer the value to stay close to the rule in that case.